pkg/services/librarypanels/librarypanels.go

```diff
@@ -47,7 +53,7 @@ func (pl *LibraryPanelService) AddMigration(mg *migrator.Migrator) {
 			{Name: "org_id", Type: migrator.DB_BigInt, Nullable: false},
 			{Name: "folder_id", Type: migrator.DB_BigInt, Nullable: false},
 			{Name: "title", Type: migrator.DB_NVarchar, Length: 255, Nullable: false},
-			{Name: "data", Type: migrator.DB_Text, Nullable: false},
+			{Name: "model", Type: migrator.DB_Text, Nullable: false},
```

Comment:
Won't renaming this column in an existing migration cause a bug?

Reply:
@papagian Do you know if changing this migration's column will cause a bug?

Reply:
this is a new table

Reply:
It's not big problem here because this is a new feature not yet released but if it can be a problem in development installations where this migration has already been executed because for those installations `model` column wouldn't exist.
Therefore is better avoiding doing that. Alternatively a new `NewAddColumnMigration` migration for `model` column can be introduced. More detailed guidelines for database migrations you can refer [here](https://github.com/grafana/grafana/blob/master/contribute/architecture/database.md#migrations).

Reply:
@torkelo It's not really a new table though, since the migration is already in master. I guess it's behind a feature flag OTOH, and can cause the problems @papagian mentions. So yeah I would agree with @papagian and say it's best to not take this risk.

Reply:
no dev installations have this feature enabled yet

Reply:
it's behind a new feature toggle that only people working on this feature (hugo) has enabled

Reply:
OK, if we can be sure it doesn't affect any deployments, it should be safe.

Reply:
Since it's early days for this feature we decided to put the db migration behind the feature toggle as well so we can make db changes without migrations during this initial phase, of course as more developers start testing/using it we need to start using db migrations

Reply:
This was a deliberate change from my side and I figured that it wouldn't impact anyone (behind feature toggle and new table) but thank you for clarifying the process moving forward.